daml-assistant/integration-tests/run-test.sh

```diff
@@ -1,15 +1,21 @@
-#!/bin/bash
+#!/usr/bin/env bash
 # Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 set -euxo pipefail
+
+RELEASE_TARBALL=$1
 TMP_DIR=$(mktemp -d)
-tar xf /data/sdk-release-tarball.tar.gz --strip-components=1 -C "$TMP_DIR"
-export PATH=~/.daml/bin:$PATH
+QUICKSTART_DIR=$(mktemp -u -d)
+export DAML_HOME=$(mktemp -u -d)
+export PATH="$DAML_HOME/bin":$PATH
+
+tar xf "$RELEASE_TARBALL" --strip-components=1 -C "$TMP_DIR"
 "$TMP_DIR"/install.sh
```

Comment:
I'm not sure what global state this might mutate, though.

Reply:
Nothing, it only writes to `DAML_HOME`.